### PR TITLE
Ensuring the scaling config serialized to numbers.

### PR DESCRIPTION
**Pull request by varikin**

### CODE REVIEW

**Comment by varikin** [comment]
@spinnaker/reviewers PTAL

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy [hunk 1]
```diff
@@ -243,22 +244,26 @@ class OpenstackServerGroupCachingAgent extends AbstractOpenstackCachingAgent imp
...
+                                      adjustment: getInteger(stack.parameters.get("scale${it}_adjustment".toString()), null),
+                                      threshold : getInteger(stack.parameters.get("scale${it}_threshold".toString()), null)]
         result.put("scale$it".toString(), scaler)
       }
     }
 
     result
   }
 
+  Integer getInteger(String value, Integer defaultValue = 0) {
```
Comment (duftler) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy]: I didn't follow this so thoroughly, but seems like the same thing that this method is doing can be accomplished via `as Integer`.

Comment (varikin) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/agent/OpenstackServerGroupCachingAgent.groovy]: Probably...I forget about that

**Comment by lwander** [comment]
LGTM, needs a rebase

**Comment by varikin** [comment]
Rebased.

**Pull request merged**